README.md

```diff
@@ -1020,6 +1020,7 @@ Utilities for managing game servers.
 - [ARRCON](https://github.com/radj307/ARRCON) - Terminal-based RCON client compatible with any game servers using the Source RCON Protocol. `GPL-3.0` `C++`
 - [Lancache](https://lancache.net) `⚠` - LAN Party game caching made easy. ([Source Code](https://github.com/lancachenet/monolithic)) `MIT` `Docker/Shell`
 - [LinuxGSM](https://linuxgsm.com/) - CLI tool for deployment and management of dedicated game servers on Linux: more than 120 games are supported. ([Source Code](https://github.com/GameServerManagers/LinuxGSM)) `MIT` `Shell`
+- [PufferPanel](https://www.pufferpanel.com/) - Web-based management system for multiple game servers. You can give other users access to your servers or create their own ([Source Code](https://github.com/pufferpanel/pufferpanel)) `Apache-2.0` `GO`
```

Comment:
```suggestion
- [PufferPanel](https://www.pufferpanel.com/) - PufferPanel is an open source game server management panel, designed for both small networks and game server providers. ([Source Code](https://github.com/pufferpanel/pufferpanel)) `Apache-2.0` `Go`
```

Reply:
As nodiscc has suggested, once this description is adjusted I believe this shall be fit for merging.